Update to 2025 version (#27)
Ferrite-FEM/FerriteCon: 2025/index.md

+++
title = "Ferrite.jl Conference 2025"
hascode = false
hasmath = false
+++

# FerriteCon 2025
FerriteCon 2025 will take place at [Technical University of Denmark (DTU)](https://www.dtu.dk/english/) 
outside Copenhagen, Denmark, on August 28th, 2025, supported by
[The Danish Center for Applied Mathematics and Mechanics, DCAMM](http://www.dcamm.dk).

## Ferrite.jl User & Developer Conference 2025
[Ferrite.jl](https://ferrite-fem.github.io/Ferrite.jl/stable/) is an open-source
finite element package written in the [Julia programming
language](https://julialang.org/). On Thursday the 28th of August 2025 we will host the fourth annual

> Ferrite.jl user and developer conference at the Technical University of Denmark (DTU) in Lungby, Denmark.

Reviewer: ```suggestion
Ferrite.jl user and developer conference at the Technical University of Denmark (DTU) in Lyngby, Denmark.
```

[registration](#registration_and_abstract_submission) is necessary.

If you have experience with Ferrite.jl we also encourage you to contribute to
the conference by giving a presentation. Here are some suggested topics that
would fit well:

 - Present your research and how you use Ferrite.jl (take this opportunity to
   discuss implementation aspects which are normally not discussed much on
   regular scientific conferences!)
 - Present interesting synergies between Ferrite.jl and the rest of the Julia
   package ecosystem
 - Present how you use Ferrite.jl for teaching or for student projects
 - Describe what you would like to use Ferrite.jl for, and the reasons why you
   are currently using something else.
 - Present ideas for future Ferrite.jl improvements and describe how you would
   like to incorporate it (perhaps describe prior art from other software
   packages)

If you have another topic you would like to discuss, please feel free to do so,
as long as you can relate it to Ferrite.jl!

## Registration and abstract submission
To register without a presentation, please register as soon as possible, but latest **15th of August (2025-08-15)**,

> by sending an email with name and affiliation to [**TBD**](mailto:xxx@xxx.com).

Reviewer: ```suggestion
by sending an email with name and affiliation to [Kim Louisa Auth](mailto:kloau@dtu.dk).
```

To register as a speaker, please submit a title and a brief abstract (~100 words, 3-4 sentences) to *TBD*

> [**TBD**](mailto:xxx@xxx.com) by the **31th of July (2025-07-31)**.

Reviewer: ```suggestion
[Kim Louisa Ath](mailto:kloau@dtu.dk) by the **31th of July (2025-07-31)**.
```
